civitai-downloader-v2/src/data/schema_manager.py

```
@@ -0,0 +1,375 @@
+#!/usr/bin/env python3
+"""
+Database Schema Manager - Centralized database schema initialization.
+Eliminates code duplication across multiple components that need database initialization.
+"""
+
+import sqlite3
+import logging
+from typing import Dict, List, Optional, Callable
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)